pcsx2/gui/Debugger/CtrlMemView.cpp

```diff
@@ -125,137 +138,167 @@ void CtrlMemView::redraw()
 	render(dc);
 }
 
+int CtrlMemView::hexGroupPositionFromIndex(int idx)
+{
+	int groupPos = idx * charWidth * 2;
+
+	int space = (charWidth / 4);
+
+	// spaces after every byte
+	groupPos += idx * space;
+
+	// spaces after every 2 bytes
+	groupPos += (idx / 2) * space;
+
+	// spaces after every 4 bytes
+	return groupPos + (idx / 4) * space;
+}
+
 void CtrlMemView::render(wxDC& dc)
 {
 	bool hasFocus = wxWindow::FindFocus() == this;
-	int visibleRows = GetClientSize().y/rowHeight;
+	int visibleRows = GetClientSize().y / rowHeight;
 
-	wxColor white = wxColor(0xFFFFFFFF);
-	dc.SetBrush(wxBrush(white));
-	dc.SetPen(wxPen(white));
+	static wxColor COLOR_WHITE = wxColor(0xFFFFFFFF);
```

Comment:
Mark the variable as const rather than static. Static will use extra memory for nothing.